doc/sphinx/Pacemaker_Explained/status.rst

```diff
@@ -1,375 +1,374 @@
+.. index::
+   single: status
```

Comment:
Let's also add single: XML element, status

I'm thinking of having a complete index of XML elements used in the CIB. We could maybe do XML attributes as well, though I didn't do that for the alerts chapter -- maybe for the future